jujugui/static/gui/src/app/components/profile/header/header.js

```diff
@@ -148,20 +148,20 @@ class ProfileHeader extends React.Component {
         </li>
       ];
     }
-    return <ul className="p-list ts-profile-header__menu">{items}</ul>;
+    return <ul className="p-list u-no-margin--bottom ts-profile-header__menu">{items}</ul>;
   }
 
   render() {
     const user = this.state.user || {};
     return (
-      <div className="profile-header v1">
```

Comment:
v1 isnt' required anymore?

Reply:
It's moved to the parent container so no longer needed here. I hate having to use them at all but two Vanillas so 🤷🏻‍♂️